### PR TITLE
added hindi language translations

### PR DESCRIPTION
**Pull request by fadkeabhi**
#1

### CODE REVIEW

**Review by arthurits — APPROVED** [review]

**Review by arthurits — APPROVED** [review]

**Review by arthurits — APPROVED** [review]

File: SignalAnalysis/localization/strings.hi.resx [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+    <value>डेटा सहेजा गया</value>
+  </data>
+  <data name="strMsgBoxTaskCancel" xml:space="preserve">
+हॉसडॉर्फ-बेसिकोविच फ्रैक्टल आयाम की गणना रोक दी गई है।</value>
+  </data>
+  <data name="strMsgBoxTaskCancelTitle" xml:space="preserve">
+    <value>विराम</value>
+  </data>
+  <data name="strOpenDlgFilter" xml:space="preserve">
+    <value>ErgoLux files (*.elux)|*.elux|SignalAnalysis files (*.sig)|*.sig|Text files (*.txt)|*.txt|Binary files (*.bin)|*.bin|All files (*.*)|*.*</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.hi.resx]: Is this OK in Hindi?

File: SignalAnalysis/localization/strings.hi.resx [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+    <value>गलती</value>
+  </data>
+  <data name="strReadNotimplementedError" xml:space="preserve">
+    <value>किसी {0} फ़ाइल से डेटा पढ़ने में असमर्थ।</value>
+  </data>
+  <data name="strReadNotimplementedErrorTitle" xml:space="preserve">
+    <value>लागू नहीं किया गया</value>
+  </data>
+  <data name="strSaveDlgFilter" xml:space="preserve">
+    <value>Text file (*.txt)|*.txt|SignalAnalysis file (*.sig)|*.sig|Binary file (*.bin)|*.bin|Results file (*.results)|*.results|All files (*.*)|*.*</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.hi.resx]: Is this OK in Hindi?

**Review by arthurits — CHANGES_REQUESTED** [review]
Hi @fadkeabhi. Please check the comments. Two of the strings remain in plain in English. I'm not sure whether this is all right or not. Could you please have a look at them?

File: SignalAnalysis/localization/strings.hi.resx (hunk 1 above)
Comment (fadkeabhi) [reply SignalAnalysis/localization/strings.hi.resx]: Those are file types and extensions. In my opinion It's better that they remain in English.

File: SignalAnalysis/localization/strings.hi.resx (hunk 2 above)
Comment (fadkeabhi) [reply SignalAnalysis/localization/strings.hi.resx]: Those are file types and extensions. In my opinion It's better that they remain in English.

Comment (arthurits) [reply SignalAnalysis/localization/strings.hi.resx]: All right.

**Review by arthurits — APPROVED** [review]

**Comment by arthurits** [comment]
Thanks for the Hindi translation @fadkeabhi!

File: SignalAnalysis/localization/strings.hi.resx (hunk 1 above)
Comment (arthurits) [reply SignalAnalysis/localization/strings.hi.resx]: All right.

**Pull request merged**